### ISSUE TITLE
Warning in PHP 8

### ISSUE THREAD

**Issue by ngosang** [wallabag/wallabag#6061]
* **Version**: Wallabag 2.5.2
* **PHP version**: Php 8.0
* **OS**: Linux

Warning:
```
2022/11/06 10:12:57 [error] 54#54: *1847 FastCGI sent in stderr: \"PHP message: PHP Warning:  Private methods cannot be final as they are never overridden by other classes in /var/www/wallabag/vendor/hoa/stream/Stream.php on line 195\" while reading response header from upstream, client: 172.18.0.43, server: _, request: \"POST /api/entries.json HTTP/1.1\", upstream: \"fastcgi://127.0.0.1:9000\", host: \"wallabag.aaaaa.aaaa\"\n
```

Errors:
```
"2022/11/04 16:17:15 [error] 49#49: *13893 FastCGI sent in stderr: \"PHP message: PHP Fatal error:  Allowed memory size of 134217728 bytes exhausted (tried to allocate 10505595 bytes) in /var/www/wallabag/vendor/symfony/symfony/src/Symfony/Component/HttpFoundation/Response.php on line 361\" while reading response header from upstream, client: 172.18.0.30, server: _, request: \"GET /api/entries.json?access_token=cccc&sort=created&order=desc&page=1&perPage=300&tags=&since=0 HTTP/1.1\", upstream: \"fastcgi://127.0.0.1:9000\", host: \"wallabag.aaaa.top\"\n"
"2022/11/04 16:17:15 [error] 49#49: *13893 FastCGI sent in stderr: \"PHP message: PHP Fatal error:  ob_start(): Cannot use output buffering in output buffering display handlers in /var/www/wallabag/vendor/twig/twig/src/Template.php on line 387\" while reading upstream, client: 172.18.0.30, server: _, request: \"GET /api/entries.json?access_token=ccccc&sort=created&order=desc&page=1&perPage=300&tags=&since=0 HTTP/1.1\", upstream: \"fastcgi://127.0.0.1:9000\", host: \"wallabag.aaaaa.top\"\n"
```

**Comment by nicosomb** [comment]
@ngosang can you try on php 8.1 please?

**Comment by nicosomb** [comment]
@ngosang do you know when you'll be able to test 2.6.x?

**Comment by ngosang** [comment]
It's fixed in 2.6.1
